mod/groups/start.php

```diff
@@ -496,34 +496,45 @@ function groups_create_event_listener($event, $object_type, $object) {
  * Return the write access for the current group if the user has write access to it.
  */
 function groups_write_acl_plugin_hook($hook, $entity_type, $returnvalue, $params) {
-	$page_owner = elgg_get_page_owner_entity();
-	$user_guid = $params['user_id'];
+	
+	if (empty($params) || !is_array($params)) {
```

Comment:
what if at some point core changes? what if someone triggers the same hook but forgot the params?

This will make more robust code, which core could definitely use